api/src/main/java/org/openmrs/DrugOrder.java

```diff
@@ -14,58 +14,60 @@
 
 /**
  * DrugOrder
- * 
+ *
  * @version 1.0
  */
 public class DrugOrder extends Order implements java.io.Serializable {
-	
```

Comment:
You need build the project before committing then these format changes should get weeded out